internal/command/jsonformat/jsondiff/types.go

```diff
@@ -16,11 +19,11 @@ const (
 	Null   Type = "null"
 )
 
-func GetType(json interface{}) Type {
-	switch json.(type) {
+func GetType(value interface{}) Type {
+	switch value.(type) {
 	case []interface{}:
 		return Array
-	case float64:
+	case json.Number:
 		return Number
```

Comment:
I think it may be a good idea to maintain the `float64` case here just incase we didn't catch everything.